extension/e2e-tests/helpers/login.ts

```diff
@@ -22,7 +22,7 @@ export const loginAndFund = async ({ page, extensionId }) => {
   });
 
   await page.goto(`chrome-extension://${extensionId}/index.html#/account`);
-  await page.getByTestId("BlockaidAnnouncement__accept").click();
+  // await page.getByTestId("BlockaidAnnouncement__accept").click();
```

Comment:
This isn't shown right now and also commented out so I assume we will need this again soon.